man/profile.doc

```diff
@@ -36,10 +47,29 @@ to show_profile/1.
 	\termitem{time}{+Which}
     If \arg{Which} is \const{cpu} (default), collect CPU timing
     statistics.  If \const{wall}, collect wall time statistics
-    based on a 5 millisecond sampling rate.  Wall time statistics
+    based on the specified sampling rate (see below).  Wall time statistics
     can be useful if \arg{Goal} calls blocking system calls.
     \end{description}
 
+    \begin{description}
+	\termitem{ports}{+Bool}
+    If \const{true} counts on all ports will be accumulated at the possible
+    expense of added memory overhead. If \const{false} only the \emph{call} port
+    counts are accumulated but memory overhead is reduced in some situations.
+    If neither value is specified, the default is the "classic" view which
```

Comment:
Is the "classic" view different from specifying either `ports(true)` or `ports(false)`? If so, I suggest adding a `ports(classic)` option.